bin/data-get.js

```diff
@@ -58,6 +58,9 @@ const run = async () => {
       console.log(`Time elapsed: ${(end / 1000).toFixed(2)} s`)
       console.log(`Dataset/file is saved in "${savedPath}"`)
     } else {
+      if (parsedIdentifier.type === "github" && parsedIdentifier.name.slice((parsedIdentifier.name.lastIndexOf(".") - 1 >>> 0) + 2) !== '') {
```

Comment:
Same here - not clear what you're doing in the second part of if statement. Please, add comments in the code.

Reply:
I am checking for the file extension, so if it recognises as a file, it will not pass as a dataset.

```
console.log(getFileExtension3(''));                            // ''
console.log(getFileExtension3('filename'));                    // ''
console.log(getFileExtension3('filename.txt'));                // 'txt'
console.log(getFileExtension3('.hiddenfile'));                 // ''
console.log(getFileExtension3('filename.with.many.dots.ext')); // 'ext'
```